lib/modules/versioning/cargo/index.ts

```diff
@@ -94,6 +94,10 @@ function getNewValue({
   if (!currentValue || currentValue === '*') {
     return rangeStrategy === 'pin' ? `=${newVersion}` : currentValue;
   }
+  // If the current value is a simple version, bump to fully specified newVersion
+  if (rangeStrategy === 'bump' && currentValue.match(regEx(/^(\d+\.?)+$/))) {
```

Comment:
```suggestion
  if (rangeStrategy === 'bump' && currentValue.match(regEx(/^\d+(?:\.\d+)*$/))) {
```
this should be better